client/asset/eth/eth.go

```diff
@@ -2291,9 +2305,15 @@ type findRedemptionRequest struct {
 
 // sendFindRedemptionResult sends the result or logs a message if it cannot be
 // sent.
-func (eth *baseWallet) sendFindRedemptionResult(req *findRedemptionRequest, secretHash [32]byte, secret []byte, err error) {
+func (eth *baseWallet) sendFindRedemptionResult(
```

Comment:
You could split this into 2 lines instead of 1 line per arg. That's what I see elsewhere in the codebase.

Reply:
Ok, applied.